ACM_General/sigs/tests.py

```diff
@@ -62,6 +87,13 @@ def test_sig_model_member_functions(self):
         self.assertEqual(str(sig), sig.id)
 
     def test_sig_unique_constraint(self):
+        """
+        Verifies that you will get an error is trying to make two of the
```

Comment:
Verifies the uniqueness constraint for SIG creation (try to avoid using personal pronouns)